ocp_resources/resource.py

```diff
@@ -1086,3 +1092,15 @@ def _apply_patches(patches, action_text, action):
                 resource.update_replace(
                     resource_dict=patch
                 )  # replace the resource metadata
+
+    @staticmethod
```

Comment:
`@staticmethod` should be removed and `_apply_patches` should be called with self.
`func=self._apply_patches`

Reply:
ok. I will update this